### DISCUSSION TITLE
TCA quadrant bindings IGN Start

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#7145] [General]
Hi,

is it possible that the bindings for the TCA quadrant changed? I have problems when switching the IGN knob to IGN/START. He is always jumping back to NORM MODE?! My last working bindings like this:

![grafik](https://user-images.githubusercontent.com/76531893/166222903-b9cc0389-6616-4d6a-9cf0-3b1b801dcf34.png)

greets from germany

**Comment by tracernz** [comment]
Something broke on the MSFS side with sim update 9. There is a way to re-bind it to work but I haven't done it myself yet. https://www.youtube.com/watch?v=gnVyTFr0G8Q

**Reply by CptBayer** [reply]
thanx .... I will try ...

ok ... does it like in the video. As well the settings with "on press" and "on release". Turning into IGN START position is working. But when try to turn itback to NORM MODE, no movement of the switch.... I will check it after a restart and will report ...

greets from germany
